docs/setup/metrics.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Metrics Anomaly Detection (beta)
+---
+
+Metrics anomaly detection in Opni depends on having a Prometheus instance set up, and collecting metrics from the Kubernetes cluster.  The Opni metrics service exposes the predictions via a Prometheus metrics enddpoint so this also needs to be regularly scraped.
+
+The simplest way to set up Prometheus is to use [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator).  This can be installed with the [community helm chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack), or using the Rancher Monitoring application.
```

Comment:
Should we link to https://github.com/prometheus-operator/kube-prometheus? I believe this is the most widely recommended installation  method.